src/custom/components/CurrencyInputPanel/CurrencyInputPanelMod.tsx

```diff
@@ -148,32 +148,6 @@ export const StyledBalanceMax = styled.button`
     margin-right: 0.5rem;
   `};
 `
-export const AuxInformationContainer = styled(Container)<{
```

Comment:
Wasn't part of the original UNI code (AFAIKT), thus moved to the `index` file to apply further mods.